repos/fdbt-site/src/interfaces/index.ts

```diff
@@ -736,9 +755,14 @@ export interface CapDetails {
 }
 
 export interface CapInfo {
+    id?: number;
     cap: Cap;
     capStart?: CapStartInfo;
 }
+
+export interface CapSelection {
+    id: number | null;
```

Comment:
I dont think we need the | null here - if there is a cap selection, the id will be a number. If its not a number, there is no cap selection.